### PR TITLE
Megaman X8 16-bit

### PR DESCRIPTION
**Pull request by Damian2809**
# New Port for { Megaman X8 16-bit }

## Game Information
- **Title**: { MegaMan X8 16-bit }
- **URL**: { https://sonicfangameshq.com/forums/showcase/mega-man-x8-16-bit-demo3.1504/ }

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [X] AmberELEC
- [X] ArkOS
- [X] ROCKNIX
- [X] muOS

### Resolution Tests
Test all major resolutions:
- [X] 480x320
- [X] 640x480
- [X TESTED BUT NOT VERY GOOD] 720x720 (RGB30)
- [X] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/megamanx816-bit/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+## Notes
+
+https://sonicfangameshq.com/forums/showcase/mega-man-x8-16-bit-demo3.1504/ grab the zip from here, extract it and rename the File to "Megaman X8 FC.pck" after that put it in megamanx816-bit/gamedata folder
```
Comment (Cebion) [inline ports/megamanx816-bit/README.md:3]: Notes section is for thank you notes with links to the developer etc. 
Instructions go into port.json

File: ports/megamanx816-bit/cover.png
Comment (Cebion) [inline ports/megamanx816-bit/cover.png:1]: If a cover is used has to show also gameplay not just the logo.
Else just use a screenshot.
If using a cover / screenshot has to be 640x480 in resolution.

File: ports/megamanx816-bit/port.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+  "version": 3,
+  "name": "megamanx8.zip",
```
Comment (Cebion) [inline ports/megamanx816-bit/port.json:3]: zip name and folder name have to match

File: ports/megamanx816-bit/screenshot.png
Comment (Cebion) [inline ports/megamanx816-bit/screenshot.png:1]: screenshot has to be 640x480, and no collage

**Review by Cebion — CHANGES_REQUESTED** [review]

File: ports/megamanx816-bit/port.json (hunk 2 above)
Comment (Damian2809) [reply ports/megamanx816-bit/port.json]: that should be fixed now

File: ports/megamanx816-bit/README.md
Comment (Cebion) [inline ports/megamanx816-bit/README.md:1]: Readme is missing credits, see other README.md for reference.

**Review by Cebion — CHANGES_REQUESTED** [review]

File: ports/megamanx816-bit/README.md
Comment (Damian2809) [reply ports/megamanx816-bit/README.md:1]: done hope thats good

**Review by Slayer366 — APPROVED** [review]

File: ports/megamanx816-bit/port.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Mega Man X8 16-bit",
+    "porter": [
+      "Damian2809"
+    ],
+    "desc": "The Mega Man X8 16-Bit Demake is a fan-made project that reimagines the original Mega Man X8 in the style of the classic 16-bit era. This demake brings the modern aesthetics and mechanics of the original game into a retro environment reminiscent of the SNES era, particularly the earlier Mega Man X titles on that system.",
+    "desc_md": null,
+    "inst": "https://sonicfangameshq.com/forums/showcase/mega-man-x8-16-bit-demo3.1504/ grab the zip from here, extract it and rename the File to "Megaman X8 FC.pck" after that put it in megamanx816-bit/gamedata folder",
```
Comment (Cebion) [inline ports/megamanx816-bit/port.json:16]: Invalid JSON!
Error: Parse error on line 16:
...rename the File to "Megaman X8 FC.pck" a
-----------------------^
Expecting 'EOF', '}', ':', ',', ']', got 'undefined'

You have 3x " in there

**Review by Cebion — CHANGES_REQUESTED** [review]

File: ports/megamanx816-bit/port.json [hunk 4]
```diff
@@ -7,13 +7,13 @@
   ],
   "items_opt": [],
   "attr": {
-    "title": "Mega Man X8 16-bit",
+    "title": "Megaman X8 16-bit",
     "porter": [
       "Damian2809"
     ],
-    "desc": "The Mega Man X8 16-Bit Demake is a fan-made project that reimagines the original Mega Man X8 in the style of the classic 16-bit era. This demake brings the modern aesthetics and mechanics of the original game into a retro environment reminiscent of the SNES era, particularly the earlier Mega Man X titles on that system.",
+    "desc": "he Mega Man X8 16-Bit Demake is a fan-made project that reimagines the original Mega Man X8 in the style of the classic 16-bit era. This demake brings the modern aesthetics and mechanics of the original game into a retro environment reminiscent of the SNES era, particularly the earlier Mega Man X titles on that system.",
```
Comment (Cebion) [inline ports/megamanx816-bit/port.json:14]: Introduced a typo

**Review by Cebion — CHANGES_REQUESTED** [review]

**Review by Cebion — APPROVED** [review]

**Review by kloptops — APPROVED** [review]

**Pull request merged**